lib/node_modules/@stdlib/math/special/scripts/function_database.json

```diff
@@ -0,0 +1,92 @@
+    "output_policy": "real_floating_point",
+    "excluded_dtypes": [ "float16", "uint8c", "int64", "uint64" ],
+    "scalar_kernels": {
+      "int8": "@stdlib/math/base/special/sqrtf",
+      "int16": "@stdlib/math/base/special/sqrtf",
+      "int32": "@stdlib/math/base/special/sqrtf",
+      "uint8": "@stdlib/math/base/special/sqrtf",
+      "uint16": "@stdlib/math/base/special/sqrtf",
+      "uint32": "@stdlib/math/base/special/sqrtf",
+      "float32": "@stdlib/math/base/special/sqrtf",
+      "float64": "@stdlib/math/base/special/sqrt",
+      "generic": "@stdlib/math/base/special/sqrt"
+    }
+  },
+  "sin": {
+    "input_dtypes": "real_and_generic",
+    "output_dtypes": "real_floating_point_and_generic",
+    "output_policy": "real_floating_point",
+    "excluded_dtypes": [ "float16", "uint8c", "int64", "uint64" ],
+    "scalar_kernels": {
+      "int8": "@stdlib/math/base/special/sinf",
+      "int16": "@stdlib/math/base/special/sinf",
+      "int32": "@stdlib/math/base/special/sinf",
+      "uint8": "@stdlib/math/base/special/sinf",
+      "uint16": "@stdlib/math/base/special/sinf",
+      "uint32": "@stdlib/math/base/special/sinf",
+      "float32": "@stdlib/math/base/special/sinf",
+      "float64": "@stdlib/math/base/special/sin",
+      "generic": "@stdlib/math/base/special/sin"
+    }
+  },
+  "floor": {
+    "input_dtypes": "numeric_and_generic",
+    "output_dtypes": "numeric_and_generic",
+    "output_policy": "same",
+    "excluded_dtypes": [ "float16", "uint8c", "complex32", "int64", "uint64" ],
+    "scalar_kernels": {
+      "int8": "@stdlib/number/int8/base/identity",
+      "int16": "@stdlib/number/int16/base/identity",
+      "int32": "@stdlib/number/int32/base/identity",
+      "uint8": "@stdlib/number/uint8/base/identity",
+      "uint16": "@stdlib/number/uint16/base/identity",
+      "uint32": "@stdlib/number/uint32/base/identity",
+      "float32": "@stdlib/math/base/special/floorf",
+      "float64": "@stdlib/math/base/special/floor",
+      "complex64": "@stdlib/math/base/special/cfloorf",
+      "complex128": "@stdlib/math/base/special/cfloor",
+      "generic": "@stdlib/math/base/special/floor"
+    }
+  },
+  "fibonacci": {
+    "input_dtypes": "real_and_generic",
+    "output_dtypes": "real_and_generic",
+    "output_policy": "real_and_generic",
+    "excluded_dtypes": [ "float16", "uint8c", "int64", "uint64" ],
+    "scalar_kernels": {
+      "int8": "@stdlib/math/base/special/fibonaccif",
+      "int16": "@stdlib/math/base/special/fibonaccif",
+      "int32": "@stdlib/math/base/special/fibonaccif",
+      "uint8": "@stdlib/math/base/special/fibonaccif",
+      "uint16": "@stdlib/math/base/special/fibonaccif",
+      "uint32": "@stdlib/math/base/special/fibonaccif",
+      "float32": "@stdlib/math/base/special/fibonaccif",
+      "float64": "@stdlib/math/base/special/fibonacci",
+      "generic": "@stdlib/math/base/special/fibonacci"
+    }
+  },
+  "abs": {
```

Comment:
Let's go ahead and sort this file in alphabetical order, such that `abs` comes before `sqrt`, etc.